web/src/components/DateTimeInput.tsx

```diff
@@ -23,12 +24,25 @@ const DateTimeInput: React.FC<Props> = ({ value, onChange }) => {
       onBlur={(e) => {
         const inputValue = e.target.value;
         if (inputValue) {
-          const date = dayjs(inputValue, DATE_TIME_FORMAT, true).toDate();
+          const date = dayjs(inputValue).toDate();
           // Check if the date is valid.
           if (!isNaN(date.getTime())) {
             onChange(date);
           } else {
-            toast.error("Invalid datetime format. Use format: 12/31/2023, 23:59:59");
+            toast.error(
```

Comment:
Could we use a simple error message instead? Maybe just an example of wanted datetime.